applications/luci-app-upnp/htdocs/luci-static/resources/view/status/include/80_upnp.js

```diff
@@ -28,7 +28,7 @@ handleDelRule = function(num, ev) {
 };
 
 return baseclass.extend({
-	title: _('Active UPnP Redirects'),
+	title: _('Active UPnP IGD & PCP/NAT-PMP Port Forwards'),
```

Comment:
And BTW "forwards" or "forwardings"? @systemcrash please clarify.

Reply:
Well... many OpenWrt devs aren't native speakers :) The "Port Forwards" means "the port (itself) is forwards somethings".
On the Wiki it's called forwarding. https://en.wikipedia.org/wiki/Port_forwarding

Reply:
> Well... many OpenWrt devs aren't native speakers :) The "Port Forwards" means "the port (itself) is forwards (something)".
> On the Wiki it's called forwarding. https://en.wikipedia.org/wiki/Port_forwarding

"It" - Forwarding - is the concept, a port forward is the correct noun. 

Neither native speakers, nor grammar anywhere ever advocates for "is forwards". We have gerunds for that. The "Port Forwards" is a plural countable noun in the case of firewall. So "configure a port forward under x" or "this port is forwarded to x". It's tenuous yet possible to say "this port forwards to x", but only because it 'is configured' to do so.

@Self-Hosting-Group you're fine here.

Reply:
1. yes
2. Ruleset cleanse interval (where 0= disabled)
3. We don't support those older ones, which is why wiki generally are updated to point to the latest or last two releases
   * one page
   * up to you
   * with what package?

Reply:
Good.